### PR TITLE
State overview

### PR DESCRIPTION
**Pull request by immunity-anomaly**
implement command to get state overview of all devices. see https://github.com/gewv-tu-dresden/vosekast_control/issues/18

### CODE REVIEW

File: vosekast_control/Pump.py [hunk 1]
```diff
@@ -68,3 +68,6 @@ def state(self, new_state):
         self.logger.info(f"New state of pump {self.name} is: {new_state}")
         MQTTConnection.publish_message(
             StatusMessage('pump', self.name, new_state))
+
+    def publish_state(self):
+        MQTTConnection.publish_message(StatusMessage('pump', self.name, self.state))
```
Comment (karlTGA) [inline vosekast_control/Pump.py]: Here you use the same code twice.

File: vosekast_control/Vosekast.py [hunk 2]
```diff
@@ -307,6 +297,19 @@ def clean(self):
...
 
+    def state_overview(self):
+        # for tank in self.tanks:
+        #     MQTTConnection.publish_message(StatusMessage('tank', tank.name, tank.state))
+        # for pump in self.pumps:
+        #     MQTTConnection.publish_message(StatusMessage('pump', pump.name, pump.state))
+        # for valve in self.valves:
+        #     MQTTConnection.publish_message(StatusMessage('valve', valve.name, valve.state))
+        # for level_sensor in self.level_sensors:
+        #     MQTTConnection.publish_message(StatusMessage('level_sensor', level_sensor.name, level_sensor.state))
```
Comment (karlTGA) [inline vosekast_control/Vosekast.py]: this old code can be removed. He no longer has any reason to be.

File: vosekast_control/LevelSensor.py [hunk 3]
```diff
@@ -50,3 +51,10 @@ def clear_callbacks(self):
         self._gpio_controller.add_event_detect(
             self._pin, self._gpio_controller.BOTH, bouncetime=500
         )
+    
+    @property
+    def state(self):
+        return self.position
```
Comment (karlTGA) [inline vosekast_control/LevelSensor.py]: The position means not the open or close state. It means the position on the tank.

File: vosekast_control/Vosekast.py (hunk 2 above)
Comment (immunity-anomaly) [reply vosekast_control/Vosekast.py]: ok

File: vosekast_control/Pump.py (hunk 1 above)
Comment (immunity-anomaly) [reply vosekast_control/Pump.py]: fixed

File: vosekast_control/LevelSensor.py (hunk 3 above)
Comment (immunity-anomaly) [reply vosekast_control/LevelSensor.py]: what do you propose?

Comment (karlTGA) [reply vosekast_control/LevelSensor.py]: a simple solution is, to return the current state of the GPIO

Comment (immunity-anomaly) [reply vosekast_control/LevelSensor.py]: done

**Pull request merged**